utils.test.js

```diff
@@ -3280,6 +3280,30 @@ test("parse swift deps files", () => {
     "bom-ref": "pkg:swift/github.com/apple/swift-argument-parser@1.2.2",
     repository: { url: "https://github.com/apple/swift-argument-parser.git" }
   });
+  expect(pkgList[5]).toEqual({
+    name: "swift-http-server",
+    group: "bitbucket.org:atlassian",
+    version: "0.7.4",
+    purl: "pkg:swift/bitbucket.org:atlassian/swift-http-server@0.7.4",
```

Comment:
Not sure if `:` is allowed in the namespace for purl since it is usually separated by slash. Could we replace `:` with `/` in line 6753?

Reply:
I think we dont need to replace it,` : `after it's the port number
Actually, we dont replace it in cdxgen

Reply:
I think `bitbucket.org:atlassian` must become `bitbucket.org/atlassian` to become a valid purl. Good point about port numbers. We might also need some test cases for that.

Reply:
@JingLeiTalan, can you take a look? The rest of the PR looks good, so let's try to get this merged this week.

Reply:
@prabhu  yes, i change for that, for bitbucket ssh url is a little different than github: [https://confluence.atlassian.com/bitbucketserver0815/setting-up-ssh-port-forwarding-1319572060.html#:~:text=Bitbucket%20listens%20for%20SSH%20connections%20on%20port%207999%20by%20default.]